### PR TITLE
Ability to inject to Discord Development builds.

### PR DESCRIPTION
**Pull request by dikahdoff**
As mentioned here: #526
You can inject to the Dev builds with `npm run inject dev`

### CODE REVIEW

**Comment by geshii** [comment]
It also fixes #843
But as I said in this issue, I don't know if adding it is a good idea since this client is just a canary clone and is deprecated.

File: README.md [hunk 1]
```diff
@@ -33,17 +34,20 @@ Grab the `zip` file from [here](https://github.com/BetterDiscord/Installer/relea
...
 
-
 ## Manual Installation
 
 For normal users, installing via the installers makes the most sense. However when wanting to either develop BetterDiscord, or when the installers do not work, this option can be used.
 
 **Prerequisites**: [Git](https://git-scm.com), [Node.js](https://nodejs.org/en/) 12.x+ and the package manager [npm](https://www.npmjs.com/).
 
-1. Clone this repository `git clone https://github.com/rauenzi/BetterDiscordApp.git`
+1. Clone this repository `git clone https://github.com/dikahdoff/BetterDiscordApp.git`
```
Comment (geshii) [inline README.md:43]:
```suggestion
1. Clone this repository `git clone https://github.com/rauenzi/BetterDiscordApp.git`
```
Why do you redirect to your own repo ??? I don't understand ?

File: README.md [hunk 2]
```diff
@@ -22,8 +22,9 @@ BetterDiscord is a client modification for Discord. This allows you to add plugi
 
 # Installation
 
-## Auto Installers
+## Auto Installation
```
Comment (geshii) [inline README.md:25]:
```suggestion
## Auto Installers
```
That was fine

File: scripts/inject.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const buildPackage = require("./package");
 
 const useBdRelease = args[2] && args[2].toLowerCase() === "release";
 const releaseInput = useBdRelease ? args[3] && args[3].toLowerCase() : args[2] && args[2].toLowerCase();
-const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "ptb" ? "Discord PTB" : "Discord";
+const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "dev" ? "Discord Development" : releaseInput === "ptb" ? "Discord PTB" : "Discord";
```
Comment (geshii) [inline scripts/inject.js:10]:
```suggestion
const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "dev" ? "Discord Development" : releaseInput === "canary" ? "Discord PTB" : "Discord";
```
Discord dev is a clone of canary.

**Review by geshii — CHANGES_REQUESTED** [review]

**Comment by NathanPeake** [comment]
this would be nice to have since I use this version for testing with alts

File: scripts/inject.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const buildPackage = require("./package");
 
 const useBdRelease = args[2] && args[2].toLowerCase() === "release";
 const releaseInput = useBdRelease ? args[3] && args[3].toLowerCase() : args[2] && args[2].toLowerCase();
-const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "ptb" ? "Discord PTB" : "Discord";
+const release = releaseInput === "canary" ? "Discord Canary" : releaseInput === "dev" ? "Discord Development" : "Discord";
```
Comment (DonJTru) [inline scripts/inject.js]: Why remove PTB from injecting? you can just add "dev" as another switch, (see my comment on the pr)

File: scripts/inject.js (hunk 3 above)
Comment (DonJTru) [reply scripts/inject.js:10]: Dev is a clone of canary, I do not understand the Suggested change though because ptb is in a separate file location.

**Comment by DonJTru** [comment]
> 
> 
> It also fixes #843
> But as I said in this issue, I don't know if adding it is a good idea since this client is just a canary clone and is deprecated.

the API is deprecated but discord dev still gets pushed updates and has the feature to switch the API server from canary to stable via the .json file, IE: attached files being "canary.discord.com" and "cdn.discord.com" the old API allowed a "dev.discord.com" but they reused that sub-domain for developers.discord.com.

**Comment by geshii** [comment]
@dikahdoff can you look at our requested changes?

**Pull request closed without merging**